README.md

```diff
@@ -1,2 +1,45 @@
-# helloMicroverse
-Module 1: Hello Microverse project
+# Hello Microverse
+
+> There is Hello Microverse project in Module 1 Day 2
+
+
+## Built With
+
+- HTML, CSS
+- Visual Studio Code
+- GIT and GITHUB
+
+## Live Demo (if available)
+
+https://sergskol.github.io/helloMicroverse/
+
+
+## Getting Started
+
+Follow the link to open web page.
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@SergSkol](https://github.com/SergSkol)
+- Twitter: [@sps676](https://twitter.com/sps676)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sergiy-shkolnik-7801a53/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+Appriciated to anyone whose code was used*
+* although no, because this is purely my code)
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link to your MIT license is broken because the file is absent in your codebase. Kindy add the file containing the MIT license 👍🏽

Reply:
Thanks! Done!